src/app/news.service.ts

```diff
@@ -10,16 +10,19 @@ export class NewsService {
   private _news$ = new Subject<string>();
   private _breakingNews$ = new Subject<string>();
 
-  news$ = this._news$.pipe(storeNews());
-  breakingNews$ = this._breakingNews$.pipe(storeNews());
+  news$ = this._news$.pipe(storeNews(), shareReplay());
+  breakingNews$ = this._breakingNews$.pipe(storeNews(), shareReplay());
 
-  allNews$ = combineLatest(
-    // combineLatest will start emitting when all of it's streams will emit at least one time
-    // that is why startWith() operator is required here - we want to emit normal news
-    // even when there were no breaking news yet and vice versa
-    this.breakingNews$.pipe(startWith([])),
-    this.news$.pipe(startWith([]))
-  ).pipe(map(([breaking, news]) => [...breaking, ...news]));
+  allNews$ = merge(   //merge two streams in one
+    this.breakingNews$.pipe(),
+    this.news$.pipe()
+  ).pipe(
```

Comment:
You can replace these streams with corresponding singular streams (`_news$` and `_breakingNews$`) which emit only 1 new value at a time and then you can get rid of array extraction operation.